wrapper/src/test/java/integration/container/tests/PerformanceTest.java

```diff
@@ -279,9 +281,10 @@ private void execute_FailoverTime_SocketTimeout(int socketTimeout, int sleepDela
                 .getProxyDatabaseInfo()
                 .getInstanceEndpointSuffix());
     props.setProperty("failoverTimeoutMs", Integer.toString(PERF_FAILOVER_TIMEOUT_MS));
+    FailoverConnectionPlugin.FAILOVER_MODE.set(props, "strict-reader");
```

Comment:
this is equivalent to setting that parameter to true?

Reply:
Yes